### PR TITLE
Task3

### PR DESCRIPTION
**Pull request by poberezhniuk**
- [CLICK TO SEE DEMO](https://poberezhniuk.github.io/user-managment-app/singup)

> User list page.

### CODE REVIEW

File: src/app/edit-dialog/edit-dialog.component.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```
Comment (alexandr2134) [inline src/app/edit-dialog/edit-dialog.component.ts]: keep same code-style allover the project (in this case ' vs ")

File: src/app/message-not-autorized/message-not-autorized.component.scss [hunk 2]
```diff
@@ -0,0 +1,21 @@
+@import "../variables";
```
Comment (alexandr2134) [inline src/app/message-not-autorized/message-not-autorized.component.scss:1]: keep 1 empty line after imports

File: src/app/user-list/table/table.component.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    <tr
+      mat-row
+      *matRowDef="let row; columns: displayedColumns"
+      routerLink="/user/{{ row.id }}"
+    ></tr>
+  </table>
+
+  <mat-paginator
+    [pageSize]="5"
+    [pageSizeOptions]="[1, 5, 10, 20, 50, 100]"
```
Comment (alexandr2134) [inline src/app/user-list/table/table.component.html]: move this array to constant

Comment (poberezhniuk) [reply src/app/user-list/table/table.component.html]: done

File: src/app/message-not-autorized/message-not-autorized.component.scss (hunk 2 above)
Comment (poberezhniuk) [reply src/app/message-not-autorized/message-not-autorized.component.scss:1]: done

File: src/app/edit-dialog/edit-dialog.component.ts (hunk 1 above)
Comment (poberezhniuk) [reply src/app/edit-dialog/edit-dialog.component.ts]: done

File: src/app/user-list/table/table.component.scss [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+.mat-elevation-z8 {
+  box-shadow: none;
+}
+
+table {
+  width: 100%;
+}
+
+th[mat-header-cell] {
+  background-color: lightcyan;
```
Comment (alexandr2134) [inline src/app/user-list/table/table.component.scss]: move to variable

Comment (poberezhniuk) [reply src/app/user-list/table/table.component.scss]: move to variable $anvanced-color

File: src/app/app.component.scss [hunk 5]
```diff
@@ -0,0 +1,8 @@
+.main-container {
+  display: flex;
+  justify-content: center;
+  min-width: 60%;
+  padding: 0 1rem 1rem 1rem;
+  background-color: white;
```
Comment (alexandr2134) [inline src/app/app.component.scss]: ->

**Review by alexandr2134 — APPROVED** [review]

**Pull request merged**